packages/elements/src/item/index.ts

```diff
@@ -186,6 +182,12 @@ export class Item extends ControlElement {
     if (changedProperties.has('type')) {
       this.typeChanged();
     }
+    if (changedProperties.has('multiple')) {
+      this.multipleChanged();
+    }
+    if (changedProperties.has('selected')) {
+      this.selectedChanged();
+    }
     super.update(changedProperties);
```

Comment:
In lit.dev lifecycle there is new hook, called `willUpdate`. In majority of cases we should use that instead of `update`